bundles/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/handler/AVMFritzBaseBridgeHandler.java

```diff
@@ -140,13 +139,38 @@ public void initialize() {
         }
 
         if (configValid) {
-            this.connection = new FritzAhaWebInterface(config, this, httpClient);
+            updateStatus(ThingStatus.UNKNOWN);
+            manageConnections();
+        }
+    }
 
+    protected synchronized void manageConnections() {
+        AVMFritzBoxConfiguration config = getConfigAs(AVMFritzBoxConfiguration.class);
+        if (this.connection == null) {
+            this.connection = new FritzAhaWebInterface(config, this, httpClient);
             stopPolling();
             startPolling();
+        } else {
+            if (!getThing().getChannels().stream().filter(c -> isLinked(c.getUID())).findAny().isPresent()) {
+                // no channels are linked anymore
+                stopPolling();
```

Comment:
This method stops the whole polling job for updating all devices. We should not do that.

Reply:
Oh, you're right, thanks for noticing!